_includes/navigation.html

```diff
@@ -28,6 +28,7 @@
     <ul class="col-sm-3">
       <li>Sponsors</li>
       <li><a href="/sponsors">2016 Sponsors &hearts;</a></li>
+      <li><a href="/sponsors">2016 Coaching Companies</a></li>
```

Comment:
thank for noticing! I literally just fixed this locally, am gonna push in a sec ;)